IS7024/HBowl.cs

```diff
@@ -467,7 +467,7 @@ public override object ReadJson(JsonReader reader, Type t, object existingValue,
             var value = serializer.Deserialize<string>(reader);
             if (value == "Los Angeles (LA)")
             {
-                return MetroAreaDisplayName.LosAngelesLa;
+                return MetroAreaDisplayName.LosAngeles;
```

Comment:
This change was made so that "LosAngeles" shows up when the "DisplayName" is called for MetroArea.